src/overcooked_demo/server/static/templates/index.html

```diff
@@ -64,6 +64,8 @@
     <input type="checkbox" id="showPotential" name="showPotential">
     <label for="Use Old Dynamics">Old dynamics?</label>
     <input type="checkbox" id="oldDynamics" name="oldDynamics">
```

Comment:
I would set this to true by default. Or actually, maybe we should have the checkbox be `use v2 dynamics` and change the related documentation in the repo. I think the standard should probably just be to use the old dynamics